features/step_definitions/networking.rb

```diff
@@ -631,3 +631,13 @@
     raise "Failed to delete the docker container." unless @result[:success]
   }
 end
+
+
+Given /^the multus is enabled on the cluster$/ do
+  ensure_admin_tagged
+
+  desiredMultusReplicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:desired]
+  availableMultusReplicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:available]
+
+  raise "Multus is not running correctly!" unless desiredMultusReplicas == availableMultusReplicas
```

Comment:
How about add availableMultusReplicas != 0. eg:
`availableMultusReplicas != 0 && desiredMultusReplicas == availableMultusReplicas`

Reply:
Yeah, I think it is a good point, I have updated the step to check the available number is not 0.

Thanks.